### PR TITLE
[#7] Implement the flow inside a tab

### PR DESCRIPTION
**Pull request by jorisros**

### CODE REVIEW

File: src/Resources/public/js/pimcore/startup.js [hunk 1]
```diff
@@ -1,17 +1,72 @@
...
 
     pimcoreReady: function (params, broker) {
-        // alert("YouweDataDictionaryBundle ready!");
+        var user = pimcore.globalmanager.get('user');
+
+        if (user.isAllowed('data_flow_permission')) {
+
+            //Add item to settings menu
+            var importMenu = new Ext.Action({
+                text: t('Show current dataflow'),
```
Comment (paulovitorbal) [inline src/Resources/public/js/pimcore/startup.js]:
```suggestion
                text: t('Show Data Dictionary'),
```
The name of the package is *Data Dictionary*. All the text references should mention *Data Dictionary*. The data model is just one part of it (the first one).

File: src/Resources/public/js/pimcore/startup.js [hunk 2]
```diff
@@ -1,17 +1,72 @@
...
+    /**
+     * Creates a new tab panel with a iFrame inside
+     *
+     * @param panelId
+     * @returns {Ext.panel.Panel|Ext.dashboard.Panel|Ext.form.Panel|Ext.grid.Panel|Ext.tab.Panel|Ext.tree.Panel|*}
+     */
+    createPanel: function(panelId) {
+        return new Ext.Panel({
+            id: panelId,
+            title: t('Data flow'),
```
Comment (paulovitorbal) [inline src/Resources/public/js/pimcore/startup.js]:
```suggestion
            title: t('Data Dictionary'),
```
Same from above

File: readme.md [hunk 3]
```diff
@@ -1,6 +1,37 @@
...
+And then enable the bundle:
+
+```
+./bin/console pimcore:bundle:install DataDictionaryBundle
+```
+
+And youre done!
+## Results
+
+You can get the flow by going to the setting (gear icon) -> Show current dataflow. Then a new tab wil be opened with the flow.
```
Comment (paulovitorbal) [inline readme.md]:
```suggestion
You can get the data dictionary by going to the setting (gear icon) -> Show Current Data Dictionary. Then a new tab wil be opened with the data dictionary.
```

File: readme.md [hunk 4]
```diff
@@ -1,6 +1,37 @@
...
+
+Or you can get the flow directly going to the following url:
```
Comment (paulovitorbal) [inline readme.md]:
```suggestion
Or you can get the data dictionary directly going to the following URL: 
```

File: readme.md [hunk 5]
```diff
@@ -1,6 +1,37 @@
...
+
+And youre done!
+## Results
+
+You can get the flow by going to the setting (gear icon) -> Show current dataflow. Then a new tab wil be opened with the flow.
+
+Or you can get the flow directly going to the following url: 
+[http://<localhost>/admin/data-dictionary/](http://<localhost>/admin/data-dictionary)
+
+Remember to change the *localhost* to your own pimcore name.
```
Comment (paulovitorbal) [inline readme.md]:
```suggestion
Remember to change the *localhost* to your own pimcore host name.
```

**Review by paulovitorbal — CHANGES_REQUESTED** [review]
The name of the package is Data Dictionary. All the text references should mention Data Dictionary instead of data flow. The data flow is just one part of it (the first one).

File: readme.md (hunk 5 above)
Comment (paulovitorbal) [reply readme.md]: this one is just to resolve ambiguity

File: src/Resources/public/js/pimcore/startup.js (hunk 1 above)
Comment (jorisros) [reply src/Resources/public/js/pimcore/startup.js]: I agree but you show the flow here, the rest of the documentation is at the moment not there.

**Review by paulovitorbal — APPROVED** [review]
Thank you @jorisros, really nice!

**Pull request merged**